### PR TITLE
feat(quantic): Add card container component

### PR DESCRIPTION
**Pull request by aelfaleh**
This PR add a custom `quantic-car-container` instead of `lightning-card`

image

### CODE REVIEW

**Review by nathanlb — APPROVED** [review]
Cool beans!

**Review by jeremierobert-coveo — APPROVED** [review]
Awsome

File: packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    border: var(--lwc-borderWidthThin, 1px) solid var(--lwc-colorBorder, #dddbda); 
+    border-radius: .25rem;
+    background-clip: padding-box;
+    -webkit-box-shadow: 0 2px 2px 0 rgb(0 0 0 / 10%);
+    box-shadow: 0 2px 2px 0 rgb(0 0 0 / 10%);
+}
+
+.card header {
+    padding: .75rem 1rem 0;
+    margin: 0 0 .75rem;
```
[the remark this answers is not in the document]
Comment (aelfaleh) [reply packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.css]: you are right it's just the bottom

File: packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.card {
+    position: relative;
+    padding: var(--lwc-spacingNone, 0);
+    background: #ffff;
```
[the remark this answers is not in the document]
Comment (aelfaleh) [reply packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.css]: some 8-digits color can be written using a four-digit shorthand. Four-digit shorthand is where you combine the duplicate digits from each color component into one.
ex.
image

[the remark this answers is not in the document]
Comment (aelfaleh) [reply packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.css]: in this case , I just change it with: 
`background: var(--lwc-colorBackgroundAlt, #FFFFFF);`

File: packages/quantic/force-app/main/default/lwc/quanticRecentResultsList/quanticRecentResultsList.css [hunk 3]
```diff
@@ -2,6 +2,7 @@
   display: flex;
   flex-direction: row;
   min-width: 0;
+  color: var(--brand-text-link, #0176d3);
```
Comment (nathanlb) [inline packages/quantic/force-app/main/default/lwc/quanticRecentResultsList/quanticRecentResultsList.css:5]: Apparently this one is actually camelcase
```suggestion
  color: var(--lwc-brandTextLink, #0176d3);
```
I have no idea why some are snake and some are camel. It's a zoo ¯\_(ツ)_/¯

File: packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeremierobert-coveo) [reply packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.css]: True the alpha I forgot 🤦🏻

File: packages/quantic/force-app/main/default/lwc/quanticRecentResultsList/quanticRecentResultsList.css (hunk 3 above)
Comment (jeremierobert-coveo) [reply packages/quantic/force-app/main/default/lwc/quanticRecentResultsList/quanticRecentResultsList.css:5]: `--lwc-brandTextLink` keebab-case plus camelCase 😱

File: packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html [hunk 4]
```diff
@@ -1,23 +1,27 @@
 <template>
     <template if:true={hasParentsOrValues}>
-        <lightning-card title={label}>
-            <ul class="slds-has-dividers_around-space slds-m-bottom_medium">
+        <c-quantic-card-container title={label}>
+            <ul class="slds-has-dividers_around-space slds-var-m-bottom_medium">
                 <template if:true={hasParents}>
-                    <lightning-button variant="base" label={labels.clear} title={labels.clear} icon-name="utility:clear"
-                                      class="slds-m-left_medium" onclick={reset}></lightning-button>
+                    <lightning-button
```
Comment (lbergeron) [inline packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html:6]: fix indent

**Review by lbergeron — APPROVED** [review]
Very nice!

**Pull request merged**